README.md

```diff
@@ -4,12 +4,19 @@ Ballerina IO Library
   [![Build](https://github.com/ballerina-platform/module-ballerina-io/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-io/actions?query=workflow%3ABuild)
   [![Daily build](https://github.com/ballerina-platform/module-ballerina-io/workflows/Daily%20build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-io/actions?query=workflow%3A%22Daily+build%22)
```

Comment:
```suggestion
```

We can remove daily-build workflow as well.

Reply:
Completely removed in here https://github.com/ballerina-platform/module-ballerina-io/pull/28/commits/a6f1b7fb72acf24af5fc758d3d731343e68f96c2